examples/solenoid/solenoid.madx

```diff
@@ -1,4 +1,4 @@
-beam, particle=proton, energy=0.250;
+beam, particle=proton, energy=1.188272;
```

Comment:
Fix: Added proton rest mass to MAD-X file.

Reply:
I take it this is the bug that you mentioned.  Here "energy" was modified to give the total energy in GeV (as required by MAD-X).

Reply:
That's it, yes :)